docs/components/secnav/secnav-demo.js

```diff
@@ -1,35 +1,35 @@
 import Util from '../../_util';
 
 if (document.getElementById('vue-secNavDemo')) {
-    new Vue({
-        el: '#vue-secNavDemo',
-        data: {
-            isDisabled: true,
-            notifications: 'Notifications',
-            tickets: 'Tickets',
-            support: 'Support',
-            billing: 'Billing',
-            account: 'Account',
-            user: 'User',
-            currentTab: 0,
-        },
-        methods: {
-            notify () {
-                alert('Your notifications...');
-            },
-        },
-        computed: {
-            attrDisabled: function () {
-                return (this.isDisabled ? 'disabled' : '');
-            },
-            snippet: function () {
-                return Util.snippet(`
+  new Vue({
+    el: '#vue-secNavDemo',
+    data: {
+      isDisabled: true,
+      notifications: 'Notifications',
+      tickets: 'Tickets',
+      support: 'Support',
+      billing: 'Billing',
+      account: 'Account',
+      user: 'User',
+      currentTab: 0,
+    },
+    methods: {
+      notify() {
+        alert('Your notifications...');
+      },
+    },
+    computed: {
+      attrDisabled: function () {
+        return (this.isDisabled ? 'disabled' : '');
+      },
+      snippet: function () {
+        return Util.snippet(`
                 <nav id="hxTopNav">
                 <a class="hxTopNavLogo" href="#">
                   <img src="images/helix-logo.svg" alt="Brand Logo" />
                 </a>
                 <div class="hxTopNavMenu hxTopNavOptionMenu">
-                  <hx-disclosure aria-controls="topnav-menu-options">
+                  <hx-disclosure aria-controls="topnav-menu-options" aria-expanded="true">
```

Comment:
Please remove the `aria-expanded` attribute, since this is the demo for Secondary Navigation.

Reply:
Changes have been done, please review.